docs/source/features/packaging_output_models.md

```diff
@@ -21,9 +17,6 @@ Zipfile packaging will generate a ZIP file which includes 3 folders: `CandidateM
 
 Each `BestCandidateModel` folder will include model file/folder. The folder also includes a json file which includes the Olive Pass run history configurations since input model, a json file with performance metrics and a json file for inference settings for the candidate model if the candidate model is an ONNX model.
 
-#### SampleCode
-Olive will only provide sample codes for ONNX model. Sample code supports 3 different programming languages: `C++`, `C#` and `Python`. And a code snippet introducing how to use Olive output artifacts to inference candidate model with recommended inference configurations.
```

Comment:
If OrtPerfTuning pass was run, then we need a way to let users know the recommended inference configurations.

Reply:
Would writing the recommended inference configurations in an output txt file would make sense?

Reply:
We will still generate an `inference_config.json` file for the recommended inference configuration. Do you think we should still have a sample code for how to use this?

Reply:
Then it should be sufficient.